prowler/compliance/aws/aws_account_security_onboarding_aws.json

```diff
@@ -0,0 +1,445 @@
+          },
+          {
+            "Section": "QA",
+            "Name": "QA"
+          }
+        ],
+        "Checks": [
+          "guardduty_is_enabled",
+          "guardduty_no_high_severity_findings",
+          "route53_domains_privacy_protection_enabled",
+          "securityhub_enabled",
+          "elbv2_waf_acl_attached",
+          "apigateway_restapi_waf_acl_attached",
+          "shield_advanced_protection_in_associated_elastic_ips",
+          "shield_advanced_protection_in_classic_load_balancers",
+          "shield_advanced_protection_in_cloudfront_distributions",
+          "shield_advanced_protection_in_global_accelerators",
+          "shield_advanced_protection_in_internet_facing_load_balancers",
+          "shield_advanced_protection_in_route53_hosted_zones",
+          "ecr_repositories_scan_images_on_push_enabled",
+          "ecr_repositories_scan_vulnerabilities_in_latest_image"
+        ]
+      },
+      {
+        "Id": "Budget-Alarms",
+        "Name": "Budget-Alarms",
+        "Description": "Budget Alarms for your AWS Account.",
+        "Attributes": [
+          {
+            "Section": "Prod",
+            "Name": "Prod"
+          },
+          {
+            "Section": "QA",
+            "Name": "QA"
+          }
+        ],
+        "Checks": []
+      },
+      {
+        "Id": "IAM-Access-Analyzer",
+        "Name": "IAM access analyzer",
+        "Description": "IAM access analyzer in your AWS Account.",
+        "Attributes": [
+          {
+            "Section": "Analyzer-Regions",
+            "Name": "Create analyzers in each active regions"
+          },
+          {
+            "Section": "Verify-that-events-are-present-in-SecurityHub-aggregated-view",
+            "Name": "Verify that events are present in SecurityHub aggregated view"
+          }
+        ],
+        "Checks": [
+          "securityhub_enabled"
+        ]
+      },
+      {
+        "Id": "Macie",
+        "Name": "Macie",
+        "Description": "Macie in your AWS Account.",
+        "Attributes": [
+          {
+            "Section": "Critical-Buckets",
+            "Name": "Consider enabling for critical buckets only"
+          }
+        ],
+        "Checks": [
+          "macie_is_enabled"
+        ]
+      },
+      {
+        "Id": "Vulnerability-Scanning",
+        "Name": "Vulnerability Scanning",
+        "Description": "Vulnearbility Scanning in your AWS Account.",
+        "Attributes": [
+          {
+            "Section": "EC2-used-as-servers",
+            "Name": "Apply-existing-SPCS-based-on-OU-placement"
+          },
+          {
+            "Section": "ECR-used-as-docker-image-hub",
+            "Name": "ECR used as docker image hub"
+          }
+        ],
+        "Checks": [
+          "inspector2_findings_exist",
+          "ecr_repositories_scan_images_on_push_enabled",
+          "ecr_repositories_lifecycle_policy_enabled",
+          "ecr_repositories_scan_vulnerabilities_in_latest_image"
```

Comment:
Can you add `ecr_registry_scan_images_on_push_enabled`, please?

Reply:
Done!